src/batterychecker.h

```diff
@@ -1,32 +1,55 @@
 #ifndef BATTERY_CHECKER_H
 #define BATTERY_CHECKER_H
 
-#include <QProcess>
-#include <QString>
-#include <QDebug>
 #include <QObject>
+#include <QTimer>
+#include <wiringPiI2C.h>
 
 class BatteryChecker : public QObject
 {
     Q_OBJECT
-    Q_PROPERTY(int batteryLevel READ getBatteryLevel NOTIFY batteryLevelChanged)
+    Q_PROPERTY(float busVoltage READ getBusVoltage NOTIFY busVoltageChanged)
+    Q_PROPERTY(float shuntVoltage READ getShuntVoltage NOTIFY shuntVoltageChanged)
```

Comment:
In my opinion, only battery percentage and charging state should be passed to QML.